contivmodel/contivModel.go

```diff
@@ -3457,7 +3458,8 @@ func ValidateNetprofile(obj *Netprofile) error {
 		return errors.New("bandwidth string invalid format")
 	}
 
-	if obj.Burst > 10486 {
+        bwint64 := netutils.ConvertBandwidth(obj.Bandwidth)
```

Comment:
Where is this function? It looks like it wasn't include in the PR, but I see it in the other PR

Reply:
The function from the netutils.ConvertBandwidth and we use it directly()。The function will be convert the Bandwidth value, and the value type is int64.